content/about/feature-stages/index.md

```diff
@@ -57,11 +60,12 @@ Below is our list of existing features and their current phases. This informatio
 | [Client and Server Telemetry Reporting](/docs/concepts/policies-and-telemetry/) | Stable
 | [Service Dashboard in Grafana](/docs/tasks/telemetry/metrics/using-istio-dashboard/) | Stable
 | [Istio Component Dashboard in Grafana](/docs/tasks/telemetry/metrics/using-istio-dashboard/) | Stable
+| [Distributed Tracing to Zipkin / Jaeger](/docs/tasks/telemetry/distributed-tracing/) | Stable
```

Comment:
Zipkin causes a lot of confusion in the telemetry issues.  I think this would be better worded as `Distributed tracing using the ZipKin API to Jaeger`

thoughts @douglas-reid ?

Reply:
The difference between this line and the one below it are hard to parse out. I'd suggest collapsing and just calling it Distributed Tracing.

Reply:
Changed to just Distributed Tracing